### PR TITLE
regenerate bundle after fixing base csv

### PR DESCRIPTION
**Pull request by skattoju**

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/opdev/l5-operator
+  newName: quay.io/skattoju/opdev/l5-operator
```
Comment (yuriolisa) [inline config/manager/kustomization.yaml:15]: Is that expected?

File: config/default/manager_auth_proxy_patch.yaml [hunk 2]
```diff
@@ -20,6 +20,13 @@ spec:
         - containerPort: 8443
           protocol: TCP
           name: https
+        resources:
+          limits:
+            cpu: 200m
+            memory: 128Mi
+          requests:
```
Comment (yuriolisa) [inline config/default/manager_auth_proxy_patch.yaml:27]: Could we set the limits of our resources as QOS guaranteed?

https://github.com/opdev/l5-operator-demo/runs/6309421552?check_suite_focus=true

File: config/manifests/bases/l5-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -5,7 +5,8 @@ metadata:
     alm-examples: '[]'
     capabilities: Full Lifecycle
     certified: "false"
-    description: L5 Operator is a demo operator that aims to exhibits level 5 capabilities for web applications.
+    description: L5 Operator is a demo operator that aims to demonstrate level 5 capabilities
+      for web applications.
```
Comment (yuriolisa) [inline config/manifests/bases/l5-operator.clusterserviceversion.yaml:9]: For web applications or stateful applications? What do you think?

**Review by yuriolisa — CHANGES_REQUESTED** [review]

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (makon57) [inline config/manager/kustomization.yaml:15]: is this the right one?

**Review by makon57 — APPROVED** [review]

File: config/default/manager_auth_proxy_patch.yaml (hunk 2 above)
Comment (skattoju) [reply config/default/manager_auth_proxy_patch.yaml:27]: don't think its needed the current allocation may already be too much i think..

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (skattoju) [reply config/manager/kustomization.yaml:15]: nope bug had some extraneous environment variables set

File: config/manifests/bases/l5-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (skattoju) [reply config/manifests/bases/l5-operator.clusterserviceversion.yaml:9]: ours is a stateless webapp .. thought the description should be neutral though..

**Review by yuriolisa — APPROVED** [review]

**Pull request merged**